Source/WebGPU/WebGPU/Buffer.mm

```diff
@@ -101,7 +101,10 @@ static MTLStorageMode storageMode(bool deviceHasUnifiedMemory, WGPUBufferUsageFl
     MTLResourceOptions resourceOptions = (cpuCacheMode << MTLResourceCPUCacheModeShift) | (storageMode << MTLResourceStorageModeShift) | (hazardTrackingMode << MTLResourceHazardTrackingModeShift);
     // FIXME(PERFORMANCE): Consider returning nil instead of clamping to 1.
     // FIXME(PERFORMANCE): Suballocate multiple Buffers either from MTLHeaps or from larger MTLBuffers.
-    return [m_device newBufferWithLength:std::max(static_cast<NSUInteger>(1), length) options:resourceOptions];
+    id<MTLBuffer> buffer = [m_device newBufferWithLength:std::max(static_cast<NSUInteger>(1), length) options:resourceOptions];
+    ASSERT(buffer);
+    setOwnerWithIdentity(buffer);
```

Comment:
I don't understand, the `setOwnerWithIdentity()` function is on `Device` but I don't see a `m_device.` here?

Reply:
We are inside `Device::safeCreateBuffer`